SwiftMessageBar/SwiftMessageBar.swift

```diff
@@ -207,9 +207,10 @@ public final class SwiftMessageBar {
   public static func showMessage(withTitle title: String? = nil, message: String? = nil, type: MessageType,
                                  duration: TimeInterval = 3, dismiss: Bool = true,
                                  languageDirection: NSLocale.LanguageDirection = .unknown,
+                                 accessoryView: UIView? = nil,
                                  callback: Callback? = nil) -> UUID {
     return sharedMessageBar.showMessage(withTitle: title, message: message, type: type, duration: duration,
-                                        dismiss: dismiss, languageDirection: languageDirection, callback: callback)
+                                        dismiss: dismiss, languageDirection: languageDirection, accessoryView: accessoryView, callback: callback)
```

Comment:
Can you add a comment to the parameters list for `accessoryView` please

Reply:
I added comment please review again.